onnxruntime/core/optimizer/qdq_transformer/qdq_propagation.cc

```diff
@@ -157,41 +295,44 @@ bool QDQPropagationTransformer::PropagateDQForward(Graph& graph) const {
       continue;
     }
```

Comment:
Could IsDQSupported be used here?

Reply:
it looks like it. maybe I can add a similar IsQSupported function too

Reply:
renamed it and used it in both propagate functions